platooning_strategic/include/platoon_config.h

```diff
@@ -19,7 +19,7 @@
 #include <iostream>
 
 /**
- * \brief Stuct containing the algorithm configuration values for the YieldPluginConfig
+ * \brief Stuct containing the algorithm configuration values for the yield_pluginConfig
```

Comment:
I think this is supposed to PlatoonPluginConfig still?

Reply:
Yeah